.travis.yml

```diff
@@ -8,6 +8,7 @@ env:
   - CXX=g++-4.8
 before_script:
   - npm install grunt-cli -g # for "grunt build"
+  - grunt build
 script:
   - npm test
   - grunt build
```

Comment:
Oh actually we already have it here... this should be OK then, no?

Reply:
Sorry about that. This is fine as well.